controls/srg_ctr/SRG-APP-000026-CTR-000070.yml

```diff
@@ -7,4 +7,62 @@ controls:
   rules:
   - audit_rules_sysadmin_actions
   - audit_rules_usergroup_modification
+  check: |-
+    Verify  Red Hat Enterprise Linux CoreOS (RHCOS) generates audit records for all account creations, modifications, disabling, and termination events that affect ""/etc/shadow"".
```

Comment:
```suggestion
    Verify  Red Hat Enterprise Linux CoreOS (RHCOS) generates audit records for all account creations, modifications, disabling, and termination events that affect "/etc/shadow".
```

Reply:
oops, thanks. I was grabbing this data from the spreadsheet with a script and for some reason the script read double quotes from the spredsheet..gotta fix that. Thanks!